src/utils/index.js

```diff
@@ -35,10 +35,17 @@ export const TOO_MANY_FILES_REJECTION = {
   message: 'Too many files'
 }
 
+function fileExtensionIsAccepted(file, accept) {
```

Comment:
I think this should probably go inside  [react-dropzone/attr-accept](https://github.com/react-dropzone/attr-accept).